src/inet/UDPEndPoint.h

```diff
@@ -245,6 +247,8 @@ class DLL_EXPORT UDPEndPoint : public EndPointBasis<UDPEndPoint>
      */
     virtual void Free() = 0;
 
+    virtual inline void SetOtInstance(otInstance * instance) { (void) instance; }
```

Comment:
This feels like a loss of abstraction: openthread leaking into the generic interface.

@kpschoedel - is there some good way of approaching things?

Without knowing much about this, I would rather name this `SetNativeParams(void *params)` if we really want something generic. Saying `Ot` (please use OpenThread) makes it clear that abstractions leaked.

Reply:
Good solution, will do.

Reply:
Done